test/models/user_test.rb

```diff
@@ -584,7 +582,7 @@
             create(:activity, type_id: Activity.type_ids[activity_type], user_id: @user.id, created_at: @created_at)
           end
 
-          it "1であること" do
+          it '1であること' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.